src/dotnet-uninstall/Shared/BundleInfo/RuntimeVersion.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Utils;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo
+{
+    internal class RuntimeVersion : BundleVersion, IEquatable<RuntimeVersion>, IComparable, IComparable<RuntimeVersion>
```

Comment:
Can you help me understand the difference between bundleversion and runtimeversion here?

Reply:
`BundleVersion` is an abstract class and cannot be instantiated. It is the base class of both `SdkVersion` and `RuntimeVersion`. This way we can have for instance `List<BundleVersion>` so that both `SdkVersion` and `RuntimeVersion` can be included.

The reason I would like to distinguish SDK versions and Runtime versions is that SDK versions are of the form `{major}.{minor}.{sdkMinor}{patch}` whereas Runtime versions are semvers. Hence SDK versions require an additional field.

See https://docs.microsoft.com/en-us/dotnet/core/versions/.